ide/plugins/java-format/src/content/formats/webdriver-testng.js

```diff
@@ -0,0 +1,523 @@
+        '<menuitem label="6 spaces" value="6"/>' +
+        '<menuitem label="7 spaces" value="7"/>' +
+        '<menuitem label="8 spaces" value="8"/>' +
+        '</menupopup></menulist>' +
+        '<checkbox id="options_showSelenese" label="Show Selenese"/>';
+
+this.name = "TestNG (WebDriver)";
+this.testcaseExtension = ".java";
+this.suiteExtension = ".xml";
+this.webdriver = true;
+
+WDAPI.Driver = function() {
+  this.ref = options.receiver;
+};
+
+WDAPI.Driver.searchContext = function(locatorType, locator) {
+  var locatorString = xlateArgument(locator);
+  switch (locatorType) {
+    case 'xpath':
+      return 'By.xpath(' + locatorString + ')';
+    case 'css':
+      return 'By.cssSelector(' + locatorString + ')';
+    case 'id':
+      return 'By.id(' + locatorString + ')';
+    case 'link':
+      return 'By.linkText(' + locatorString + ')';
+    case 'name':
+      return 'By.name(' + locatorString + ')';
+    case 'tag_name':
+      return 'By.tagName(' + locatorString + ')';
+  }
+  throw 'Error: unknown strategy [' + locatorType + '] for locator [' + locator + ']';
+};
+
+WDAPI.Driver.prototype.back = function() {
+  return this.ref + ".navigate().back()";
+};
+
+WDAPI.Driver.prototype.close = function() {
+  return this.ref + ".close()";
+};
+
+WDAPI.Driver.prototype.findElement = function(locatorType, locator) {
+  return new WDAPI.Element(this.ref + ".findElement(" + WDAPI.Driver.searchContext(locatorType, locator) + ")");
+};
+
+WDAPI.Driver.prototype.findElements = function(locatorType, locator) {
+  return new WDAPI.ElementList(this.ref + ".findElements(" + WDAPI.Driver.searchContext(locatorType, locator) + ")");
+};
+
+WDAPI.Driver.prototype.getCurrentUrl = function() {
+  return this.ref + ".getCurrentUrl()";
+};
+
+WDAPI.Driver.prototype.get = function(url) {
+  if (url.length > 1 && (url.substring(1,8) == "http://" || url.substring(1,9) == "https://")) { // url is quoted
+    return this.ref + ".get(" + url + ")";
+  } else {
+    return this.ref + ".get(baseUrl + " + url + ")";
+  }
+};
+
+WDAPI.Driver.prototype.getTitle = function() {
+  return this.ref + ".getTitle()";
+};
+
+WDAPI.Driver.prototype.getAlert = function() {
+  return "closeAlertAndGetItsText()";
+};
+
+WDAPI.Driver.prototype.chooseOkOnNextConfirmation = function() {
+  return "acceptNextAlert = true";
+};
+
+WDAPI.Driver.prototype.chooseCancelOnNextConfirmation = function() {
+  return "acceptNextAlert = false";
+};
+
+WDAPI.Driver.prototype.refresh = function() {
+  return this.ref + ".navigate().refresh()";
+};
+
+WDAPI.Element = function(ref) {
+  this.ref = ref;
+};
+
+WDAPI.Element.prototype.clear = function() {
+  return this.ref + ".clear()";
+};
+
+WDAPI.Element.prototype.click = function() {
+  return this.ref + ".click()";
+};
+
+WDAPI.Element.prototype.getAttribute = function(attributeName) {
+  return this.ref + ".getAttribute(" + xlateArgument(attributeName) + ")";
+};
+
+WDAPI.Element.prototype.getText = function() {
```

Comment:
Fixed in [#7e0de8f](https://github.com/rasika90/selenium/commit/7e0de8f7253c45ace8d0fd539f4712e1f91349c1).